packages/create-cloudflare/src/common.ts

```diff
@@ -316,6 +316,7 @@ export const offerGit = async (ctx: PagesGeneratorContext) => {
 			startText: "Initializing git repo",
 			doneText: `${brandColor("initialized")} ${dim(`git`)}`,
 		});
+		ctx.c3InitializedGitRepo = true;
```

Comment:
some frameworks also initialize git on their own, if we encouter that scenario, we should consider the repo as initialized by c3 too and commit.

Reply:
I was going to open a separate issue for this, my thought was that we should try to prevent frameworks to do any git operation (we already do that but I don't think we do that across the board), and if they do it's sort of outside of the scope of C3

if we allow frameworks to do git operations and they for example generate an initial commit, do we them `--amend` with our own changes? or commit on top of them?

seems to me like trying to prevent framework clis to do any git operation and have a clear distinction between C3 and the frameworks in this regard could be the cleaner behavior here 🤔 

But it might not be that simple, not all CLIs could allow us to prevent git operations, and even if they do users could force git operations by passing args to the underlying CLIs with `--` 🤔

Reply:
Scrap the above comment 😅 

I've changed the behavior to account for git repos initialized by framework CLIs, please have a look 🙏